trunk/SUAVE/Methods/Weights/Buildups/Electric_Stopped_Rotor/empty.py

```diff
@@ -75,14 +75,15 @@ def empty(config,
 # Unpack Inputs
 #-------------------------------------------------------------------------------
 
-    rProp               = config.propulsors.network.propeller.prop_attributes.tip_radius
-    mBattery            = config.propulsors.network.battery.mass_properties.mass
-    mPayload            = config.propulsors.network.payload.mass_properties.mass
+    rPropLift           = config.propulsors.propulsor.propeller_lift.tip_radius
+    rPropThrust         = config.propulsors.propulsor.propeller_forward.tip_radius
+    mBattery            = config.propulsors.propulsor.battery.mass_properties.mass
+    mPayload            = config.propulsors.propulsor.payload.mass_properties.mass
     MTOW                = config.mass_properties.max_takeoff
-    nLiftProps          = config.propulsors.network.number_of_engines/2
-    nThrustProps        = config.propulsors.network.number_of_engines/2
-    nLiftBlades         = config.propulsors.network.propeller.prop_attributes.number_blades
-    nThrustBlades       = config.propulsors.network.propeller.prop_attributes.number_blades
+    nLiftProps          = config.propulsors.propulsor.number_of_engines_lift
```

Comment:
remove the "/2"?  (just to to double check)

Reply:
refer to above comment